BookingsApi/BookingsApi/Controllers/V1/WorkAllocationsController.cs

```diff
@@ -113,7 +113,7 @@ private async Task PublishAllocationsToServiceBus(List<VideoHearing> hearings, J
             var todaysHearing = hearings.Where(x => x.ScheduledDateTime.Date == DateTime.UtcNow.Date).ToList();
             if(todaysHearing.Count != 0)
             {
-                await eventPublisher.PublishAsync(new AllocationHearingsIntegrationEvent(todaysHearing, justiceUser));
+                await eventPublisher.PublishAsync(new HearingsAllocatedIntegrationEvent(todaysHearing, justiceUser));
```

Comment:
don't we need to update the BQS message handler too? As we're changing the message that's being published

Reply:
I have a feeling we use the judge name as part of the toast in the UI

Reply:
Hi Oli, it will be renamed. Just the first part of a trio of changes. The judge can be pulled in from the cache, it doesn't actually need to be pulled into this message